apps/playground-api/src/setups/setup.loan.token.ts

```diff
@@ -42,6 +42,7 @@ export class SetupLoanToken extends PlaygroundSetup<SetLoanToken> {
 
   async create (each: SetLoanToken): Promise<void> {
     await this.client.loan.setLoanToken(each)
+    await this.generate(1)
```

Comment:
yes.. its actually just random loan token id issue
because they were generated within a same block, the id will be randomly assigned by miner

![random-loan-token-id](https://user-images.githubusercontent.com/8097901/160962738-6dfd3477-8f86-4bc5-b543-0b11dbff7c5d.png)